source/background.ts

```diff
@@ -3,27 +3,41 @@ import {addContextMenu} from 'webext-domain-permission-toggle';
 import {addToFutureTabs} from 'webext-dynamic-content-scripts';
 import './libs/cache';
 
-// Define defaults
+export interface Options {
+	customCSS: string;
+	personalToken: string;
+	logging: boolean;
+	[featureName: string]: string | boolean;
```

Comment:
This index signature is required because we're adding a lot of `'feature:XXX': boolean` to the options, but it's causing some trouble. More info in the next comments.

cc @nickytonline

Reply:
I'll take a look later tonight.